main/src/cgeo/geocaching/connector/lc/LCApi.java

```diff
@@ -222,21 +230,19 @@ private static Geocache parseCache(final JsonNode response) {
             cache.setType(CacheType.ADVLAB);
             cache.setSize(CacheSize.getById("virtual"));
             cache.setArchived(response.get("IsArchived").asBoolean()); // we get that even in passive mode!
-            // cache.setFound(response.get("IsComplete").asBoolean()); as soon as we're using active mode
+            cache.setFound(response.get("IsComplete").asBoolean());    //as soon as we're using active mode
```

Comment:
//as soon as we're using active mode

Maybe change the comment to something more fitting for the current situation?